boards/nxp/lpcxpresso54114/doc/index.rst

```diff
@@ -170,7 +170,7 @@ Here is an example for the :ref:`hello_world` application.
 
 .. zephyr-app-commands::
    :zephyr-app: samples/hello_world
-   :board: lpcxpresso54114_m4
+   :board: lpcxpresso54114//m4
```

Comment:
```suggestion
   :board: lpcxpresso54114/lpc54114/m4
```

Reply:
While I agree with this, why do we then allow lpcxpresso54114//m4 to even work?

Reply:
User convenience, usage in-tree should be full and proper, the reasons why a SoC can be omitted are presented to users and then users are free to omit them if they want